ninja-core/src/main/java/ninja/Bootstrap.java

```diff
@@ -208,6 +209,7 @@ protected void configure() throws Exception {
             @Override
             protected void configure() {
                 bind(Ninja.class).to(ninjaClass).in(Singleton.class);
+                Multibinder.newSetBinder(binder(), ParamParser.class);
```

Comment:
I completely agree... I think I did it twice because I started this PR before the NinjaBaseModule class appeared ;-)